pkg/probe/crd_probe.go

```diff
@@ -45,6 +47,11 @@ const (
 	ServiceInstance = "serviceinstances.servicecatalog.k8s.io"
 	// ServiceBinding define the name of the ServiceBinding CRD
 	ServiceBinding = "servicebindings.servicecatalog.k8s.io"
+
+	// DelayCRDProbe - readiness/liveness probe is run each period time defined in `periodSeconds` parameter in chart
+	// if the value is to short then `delay` can postpone checking CRDs which is resource-intensive
+	// period seconds for CRDProbe will be `periodSeconds` * delay
+	DelayCRDProbe = 60
```

Comment:
maybe we can name it a little bit different? delay for me is quite connected with time and here we have contract which is saying that we should perform check action on X execution